pandas/core/indexes/interval.py

```diff
@@ -1314,6 +1242,19 @@ def is_all_dates(self) -> bool:
 
     # TODO: arithmetic operations
 
+    def _delegate_property_get(self, name, *args, **kwargs):
+        """ method delegation to the ._values """
+        prop = getattr(self._data, name)
+        return prop  # no wrapping for now
+
+    def _delegate_method(self, name, *args, **kwargs):
+        """ method delegation to the ._data """
+        method = getattr(self._data, name)
+        res = method(*args, **kwargs)
+        if is_scalar(res) or name in self._raw_inherit:
```

Comment:
comment.

can we not put these methods on Index?

Reply:
You mean remove them from IntervalIndex?  No idea.  @jschendel?

Reply:
no the delegation methods that you added could be on Index itself.

Reply:
i see.  yes, the point of this exercise is to build up to moving as much of it to to either ExtensionIndex or even Index as possible.  Doing this on II/CI before refactoring is just to limit the scope.

Reply:
its fine, but then will have to clean later